modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -39,14 +39,11 @@
  * <p>
  * An implementation of {@code ObservableValue} may support lazy evaluation,
  * which means that the value is not immediately recomputed after changes, but
- * lazily the next time the value is requested. All bindings and properties in
- * this library support lazy evaluation.
+ * lazily the next time the value is requested (see note 1 in the "Implementation Requirements").
  * <p>
  * An {@code ObservableValue} generates two types of events: change events and
- * invalidation events. A change event indicates that the value has changed.
- * Current implementing classes in JavaFX check for a change using reference
- * equality (and not object equality, {@code Object#equals(Object)}) of the value. An
- * invalidation event is generated if the current value is not valid anymore.
+ * invalidation events. A change event indicates that the value has changed
+ * (see note 2 in "Implementation Requirements"). An invalidation event is generated if the current value is not valid anymore.
```

Comment:
Minor: either add `the` before `"Implementation Requirements"` here or remove `the` from the earlier line for consistency.

Minor: as long as you are here, can you break this line between `An` and `invalidation`? In addition to wrapping a long line, it will reduce the diffs, which seems good given that there will be an associated CSR.